### DISCUSSION TITLE
Copy only the content of public folder to dist

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4692] [Q&A]
Hi there,

Is it possible to get rid of the public folder inside dist when running the build e.g I would like the content in the public folder to be placed directly under dist 

![image](https://github.com/web-infra-dev/modern.js/assets/52765572/0f33df4a-54ef-4b8d-9b49-e0bea77580df)

thanks

**Comment by chenjiahan** [comment]
There is no config to remove the `public` folder now.

You can use `output.copy` and copy any folder to the dist folder, so you do not need to use the `config/public` folder.

https://modernjs.dev/en/configure/app/output/copy.html
